MusicStore.DataAccess/MusicStoreContext.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data.Entity;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MusicStore.DataAccess
+{
+    public class MusicStoreContext: DbContext
+    {
+        public MusicStoreContext(): base("DefaultConnection")
+        {
+
+        }
+        public DbSet<User> Users { get; set; }
+        public DbSet<Album> Albums { get; set; }
+        public DbSet<Song> Songs { get; set; }
+        public DbSet<Artist> Artists { get; set; }
+        public DbSet<BoughtSong> BoughtSongs { get; set; }
+        public DbSet<Address> Addresses { get; set; }
+        public DbSet<Genre> Genres { get; set; }
+        protected override void OnModelCreating(DbModelBuilder modelBuilder)
+        {
+            modelBuilder.Entity<User>().Property(p => p.FirstName).HasMaxLength(20);
```

Comment:
Move to separate file

Reply:
Apply this to **ALL** FluentAPI. Move setting of each table to separate method (or even file, if necessary) and then cll it somewhere here